doc (profileCreation): restructure, fix

NathalieC opened the pull request:
* add summary of content
* add BDM AC mention
* restructure definition
* add chapter enable
* fix typos

NathalieC committed:
doc (profileCreation): +BDM AC, restructure, fix

add BDM AC mention
restructure definition
add chapter enable
fix typos

amottier commented on md/profileCreation.md:
```diff
@@ -2,11 +2,23 @@
-For development purposes, the Studio can **deploy** profiles onto the portal for you (_only for Performance and Efficiency editions_).
-
+### Fonctional
+Profiles are used to grant some rights to some entities of the organization (users / groups / roles / memberships).
```
I think we should simply the introduction by saying that profiles are used to grant some rights to users.
In a second sentence we can explain that user are associated with profiles either directly or by being members of groups with roles.

NathalieC replied:
ok

amottier commented on md/profileCreation.md:
```diff
@@ -2,11 +2,23 @@
+A profile is made of a name, a description, and a mapping with the organization entities. 
+When the profile is used to access a Portal menu, Portal menu entries must be defined in the profile too.  
+(When used to access a Living Application, the application menu is defined in the [application descriptor](applicationCreation.md) itself.)  
+Since Bonita 7.6.0, profiles can be managed in the studio, for easier packaging and deployment.  
```
I think we should not highlight when the change happen because when page will be the same for version 7.8 for example information will seems to be outdated

NathalieC replied:
ok

amottier commented on md/profileCreation.md:
```diff
@@ -20,23 +32,22 @@ A **file of profiles** is an XML file containing one or more profiles.
-There is a special file of profiles for default profiles: `default_profiles.xml`  
-This file can only contain default profiles and cannot be renamed or deleted.
-Only the mapping between default profiles and entities of the organization can be edited.
+Default profiles belong to the file `default_profiles.xml`. 
```
I think it would be nice to specify: "Default profiles (administrator, user...)"

NathalieC replied:
ok

NathalieC committed:
doc(profiles): implement requested changes

- say profiles are for users, then talk about groups, roles, and memberships
- remove "since 2016...." so as not to appear outdated
- recall what default profiles are

NathalieC committed:
doc(profiles): fix typo

fix typo in first row "organization"

alachambre commented on md/profileCreation.md:
```diff
@@ -2,11 +2,22 @@
+(When used to access a Living Application, the application menu is defined in the [application descriptor](applicationCreation.md) itself.)  
+
+Only users of Performance and Efficiency editions can create profiles.  
+Community and Teamwork can use the default profiles: **Administrator**, **User**, and **Process Manager**.
```
Only Admin and User for community ..

NathalieC replied:
And even Teamwork. PM is en Efficiency feature. Thanks for noticing.

NathalieC committed:
doc(profiles): fix editions for default profiles

* user and admin for all versions
* pm for efficiency and performance. This is the official voice.
later on we will run more tests to understand why the product also give permission to use PM profile to teamwork.

NathalieC committed:
doc(profile creation): remove mention to BDM AC

* in 7.6, no BDM AC. So removed the line.
